### DISCUSSION TITLE
Error de Python 3.10.2

### DISCUSSION THREAD

**Discussion by JoseCruzR** [LaunchX-InnovaccionVirtual/CursoIntroPython#97] [Q&A] [answered]
Hola buen día compañeros el día de hoy inicie y al seguir mis primeros pasos del modulo 0 me sale esto
![error 1](https://user-images.githubusercontent.com/99230979/153517315-2fa9c49f-13bc-4735-be04-a02a9e2fb38f.jpg)
Descargue la opción mas reciente de Python y ya he intentado lo que dice en algunas discusiones mas abajo Tengo Windows 10 de 64 Bits , espero me puedan ayudar 
![error 2](https://user-images.githubusercontent.com/99230979/153517537-059c556d-4645-41d9-bba8-6145e0354d5b.jpg)

**Comment by GabrielVillagran** [comment]
Hola que tal, se me ocurre que quiza allá que agregar python al path, no se si cuando lo instalaste hayas seleccionado la opción "agregar al path" ya que lo tengas instalado instala las extensiones de juoyter.
Si lo intentas me comentas si te funcionó, si no con gusto intento buscar más soluciones :)

**Reply by JoseCruzR** [reply]
Hola Gabriel gracias por responder, así es cuando instale python marque la casilla de Path actualmente estoy intentando con una version anterior de Python a ver si funciona

**Comment by Dfredude** [comment]
Hola Jose
- En primer lugar debes usar `%pip` en lugar de `pip` dentro de un notebook.
- Tambien asegurate de que tengas **_pip_** instalado desde el CLI con el comando `pip --version`
Te debe dar algo asi, sin errores
![image](https://user-images.githubusercontent.com/72330887/153519308-7a30d74e-4eb9-41e1-9ee9-6fe78d81bc5e.png)

Nos dices como te fue

**Reply by JoseCruzR** [reply]
Me dice: "pip" no se reconoce como un comando interno o externo,
programa o archivo por lotes ejecutable.

**Reply by Dfredude** [reply]
@JoseCruzR Vas a tener que reinstalar Python, asegurate de que venga con PIP. Lo que varios hicieron mejor para evitar este tipo de errores fue descargar Python desde el Microsoft Store. Solo asegurate de instalar el mas nuevo **_3.10_**

**Reply by JoseCruzR** [reply]
Creo que será mi opción desde  Store porque ya he descargado varios incluso ya me aparecía así como me dijiste lo de PIP con versión y seguía con el mismo error

**Comment by doguedogue** [comment]
Quizás debas instalarlo

`py -m ensurepip --upgrade`
ó
`python get-pip.py`

al final revisa la versión 
`pip -V`

https://pip.pypa.io/en/stable/installation/

**Reply by JoseCruzR** [reply]
Los instale compañero y sigue el mismo error pero gracias por el aporte

**Reply by XavCilia** [reply]
tenía el mismo error en mac y tuve q usar una versión anterior, se corrigió el problema con la versión 3.7.3 me parece es de las fechas del 25/08/2021 en VSC

**Comment by JoseCruzR** [comment, marked as answer]
Para todos los que tengan el mismo error lo sulucione siguiendo los pasos de este video https://youtu.be/YBFwFMxKyyc
desinstale python porque a pesar de todas las versiones ninguna me funciono incluso la de la tienda de microsoft, y en el video te instala la 3.9 5 que me funciono a la perfección donde dice que instalen la 3.7 de python ustedes ponen 3.9 quiero agradecer a todos los que se tomaron la molestia de intentar resolver mi problema cabe resaltar que es para windows desconozco para Mac me tomo como 6 horas pero por fin puedo continuar.
![20220210_200435](https://user-images.githubusercontent.com/99230979/153531709-21174edf-4955-42b4-ac7f-cfcf2b58cc72.jpg)
